elastic/kibana
# [Security Solution] Create new host flyout

## Summary

Create the Expandable Host flyout with the risk inputs panel.

![Screenshot 2023-12-22 at 16 56 12](https://github.com/elastic/kibana/assets/1490444/a1e14232-620d-4109-a1b6-04e45513e3ee)


<img src="https://github.com/elastic/kibana/assets/1490444/b7443ca0-c369-4b53-99b4-68810d4adab1" width="400" />


<img src="https://github.com/elastic/kibana/assets/1490444/961e37b0-e2bc-48d0-90f3-55226498529c" width="400" />






### What is included 
* Host panel creation
* Left risk inputs panel
* Refactor user panel component to be reused by Host
* Risk score section

### What is not included 
* Asset integration section
* Asset criticality



### How to test it?
* Enable experimental flag `newHostDetailsFlyout`
* Create alerts with `host.name` field
* Open the alerts page and click on the `host.name` field
* It should not display the expandable risk input panel
* Enable Risk engine
* make sure it generates data for the host
* Open the flyout once more and check if the expandable risk input panel opens
